skimage/morphology/convex_hull.py

```diff
@@ -47,7 +48,10 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
 
     """
     ndim = image.ndim
-
+    if np.max(image)<1:
```

Comment:
Sorry, I wasn't clear enough in my previous comment. I was saying that this statement will not work properly if user provides, for example, an array of `int8` with values `-1` and `0` only. I'd expect the function to draw a convex hull around `-1` elements, but in this case it'll say that the image is zero.
I don't like that we actually limit input dtypes to unsigned for 2D arrays several lines below, but this is another story. For 1D/3+D negatives values should matter.

How about `if np.count_nonzero(image) > 0:`?

Reply:
>I don't like that we actually limit input dtypes to unsigned for 2D arrays several lines below

Ignore this :) .
```
In [6]: x = np.array([-1, 0], dtype=np.int8)

In [7]: x.astype(np.uint8)
Out[7]: array([255,   0], dtype=uint8)
```